org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/hotspots/SecurityHotspotsHandlerServer.java

```diff
@@ -431,7 +432,9 @@ public ShowHotspotParameters(String projectKey, String hotspotKey, String server
   }
 
   private static class StatusResponse {
+    @SerializedName("ideName")
```

Comment:
This should work without annotation since name in JSON and java field are the same:
https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/SerializedName.html

Reply:
Yes, but I like to make things explicit. In case someone refactor the attribute name, it would be a breaking change of the serialized format.